cmd/cmd.go

```diff
@@ -525,6 +526,50 @@ func RunServer(_ *cobra.Command, _ []string) error {
 	return server.Serve(ln)
 }
 
+func startMacApp(client *api.Client) error {
```

Comment:
IMO a client should never start the server. It's sufficient to tell the user the server isn't available and how to start it than to start it automatically